<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AiFace</string>
  </property>
  <widget class="QWidget" name="mainWindow">
   <widget class="QLabel" name="logo_name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Bauhaus 93&quot;;</string>
    </property>
    <property name="text">
     <string>AiFace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="register_username">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-radius:5px;
	border:1px black solid;
}
QLineEdit:hover{
	background:yellow;
	padding:2px;

}</string>
    </property>
    <property name="placeholderText">
     <string>请输入手机号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="register_password">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-radius:5px;
	border:1px black solid;
}
QLineEdit:hover{
	background:yellow;
	padding:2px;

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;
border-radius:15px;
}
QPushButton:hover{
	
	background-color: rgb(0, 85, 255);
}</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="again_password">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border-radius:5px;
	border:1px black solid;
}
QLineEdit:hover{
	background:yellow;
	padding:2px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logo1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width:30px;
height:30px;
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/reg_user.png</normaloff>img/reg_user.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="logo2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width:30px;
height:30px;
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/reg_psd.png</normaloff>img/reg_psd.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="logo3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width:30px;
height:30px;
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/reg_psd.png</normaloff>img/reg_psd.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/close.png</normaloff>img/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="min_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/min.png</normaloff>img/min.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/back.png</normaloff>img/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>166</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 6pt &quot;Adobe Devanagari&quot;;
border:none;
</string>
    </property>
    <property name="text">
     <string>密码要包含中英文字母和数字</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
